rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -836,17 +851,20 @@ def samples(self):
 
         return self._samples
 
-    def wait(self, timeout=None):
+    def wait(self, timeout=None, return_loan=True):
```

Comment:
I added the `return_loan` parameter to allow using the previous behavior.

Reply:
I don't think we should change the API default behavior in a patch release, so this argument should be False by default. Applications may be relying on the current behavior, which allows accessing the previously-read data after a call to wait.